### ISSUE TITLE
Building syncthingtray from AUR fails

### ISSUE THREAD

**Issue by immanuelfodor** [Martchus/syncthingtray#24]
Hi, syncthingtray v0.8.2-1 is not building on Manjaro Linux (Arch derivative), however, all previous versions updated fine. I've tried it on another machine, and I get the same, no update is possible on the package because of this. Do you have any idea what should I do to make it work?

Octopi summary:

```
URL | https://github.com/Martchus/syncthingtray
Version | 0.8.2-1 (outdated installed version is 0.8.1-1)
Licenses | GPL
Depends On | qtutilities qt5-svg openssl desktop-file-utils xdg-utils qt5-webkit
Optional Deps | kio : KIO plugin for Syncthing actions in Dolphin [installed]plasma-workspace : Plasmoid for Plasma 5 desktop [installed]
Download Size | 0.00 Bytes
Installed Size | 1.94 MiB
Packager | Unknown Packager
Architecture | x86_64
Build Date | Cs - 07/06/2018 08:20:16
Install Reason | Explicitly installed
```

AUR build output:

```
....hundreds of lines, the end is...

50% tests passed, 1 tests failed out of 2

Total Test time (real) =  20.53 sec

The following tests FAILED:
          1 - syncthingconnector_run_tests (Failed)
Errors while running CTest
make[3]: *** [connector/CMakeFiles/check.dir/build.make:57: connector/CMakeFiles/check] Error 8
make[2]: *** [CMakeFiles/Makefile2:417: connector/CMakeFiles/check.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:424: connector/CMakeFiles/check.dir/rule] Error 2
make: *** [Makefile:292: check] Error 2
==> ERROR: A failure occurred in check().
    Aborting...
==> ERROR: Makepkg was unable to build syncthingtray.
```

**Comment by Martchus** [comment]
Just the tests aren't passing. Try again or build the package without them. Without full logs there's nothing I can do. That it took over 20 seconds means it likely ran into a timeout.

**Comment by immanuelfodor** [comment]
Thanks for the quick reply!

I searched for how to turn off tests, and I think I've found one of your comments at https://aur.archlinux.org/packages/syncthingtray where you suggested running the build with `makepkg --nocheck`. Then I searched for where to put this flag as I've only used the Octopi GUI with `yaourt` as AUR package manager on Manjaro to update syncthingtray so far. Found this comment: https://forum.manjaro.org/t/how-to-install-discord/22224/2 where they use `yay`, so I looked for if such a parameter exists for yaourt as well. It seems it does but it's not documented well: https://github.com/archlinuxfr/yaourt/issues/108#issuecomment-107018438

So I tried to run `yaourt --m-arg "--nocheck" -Sb syncthingtray` and the build was successful, and syncthingtray became up-to-date :) 

Thanks for pointing me in the right direction.
